front/tsconfig.json

```diff
@@ -53,7 +53,7 @@
     "suppressExcessPropertyErrors": false,
     "suppressImplicitAnyIndexErrors": false,
     "typeRoots": ["node_modules/@types"],
-    "types": ["vite/client"],
+    "types": ["vite/client", "node"],
```

Comment:
This exposes `node` specific types in the global scope... Is this necessary when switching to NPM?

Reply:
it was about missing types while trying to migrate successfully, just forgot to remove it, it's done